ansible/roles/node_exporter/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+go_arch: amd64
+
+node_exporter_install_path: /srv/node_exporter
+node_exporter_textfiles_path: /srv/node_exporter/textfiles
+node_exporter_collectors: diskstats,filefd,filesystem,loadavg,mdadm,meminfo,netdev,netstat,sockstat,stat,textfile,time,uname
```

Comment:
I don't think so. This was part of the playbook that @SuperQ provided for node_exporter.